libraries/lib-music-information-retrieval/PowerSpectrumGetter.h

```diff
@@ -50,9 +51,154 @@ template<typename T> struct PffftAllocator
    }
 };
 
+//! An unsigned number of floats that can span two addresses that are well
+//! aligned for pffft
+struct PffftAlignedCount
+{
+   // see source for pffft_aligned_malloc
+   static constexpr size_t ByteAlignment = 64;
```

Comment:
Is this constant valid, though? The library aligns memory regions to 64 bytes, but this is not required by the instruction sets it uses internally (SSE2/Neon).

Probably, it doesn't harm, so I won't block this PR for this reason.